util/api-request-list-json.js

```diff
@@ -0,0 +1,703 @@
+/*
+ * Wazuh app - Module for get the list of routes in API in JSON format 
+ * Copyright (C) 2018 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+export function getApiRequestJsonModel() {
```

Comment:
This should not be a function, please refactor it to export an array:

```js
export const apiRequestList = [
...
]
```

Then use it in your controller as follow:

```js
import { apiRequestList } from 'path/util/api-request-list.js'
```

And please, rename both the name of the export and the file.